axelrod/strategies/__init__.py

```diff
@@ -1 +1,16 @@
+from ..player import is_basic, is_cheater
 from ._strategies import *
+
+# `from ._strategies import *` import the collection `strategies`
+# Now import the Meta strategies. This cannot be done in _strategies
+# because it creates circular dependencies
+
+from .meta import MetaMajority, MetaMinority, MetaWinner, MetaHunter
+strategies.extend((MetaHunter, MetaMajority, MetaMinority, MetaWinner))
+
+# Distinguished strategy collections in addition to
+# `strategies` from _strategies.py
+
+basic_strategies =    [s for s in strategies if is_basic(s())]
```

Comment:
Could we include a test for it similar to the one for `is_cheater`? https://github.com/Axelrod-Python/Axelrod/blob/master/axelrod/tests/unit/test_classification.py#L54